samples/cpp/speech_sample/main.cpp

```diff
@@ -506,13 +506,42 @@ int main(int argc, char* argv[]) {
                             inferRequest.frameIndex = -1;
                             continue;
                         }
+                        ptrInputBlobs.clear();
+                        if (FLAGS_iname.empty()) {
+                            for (auto& input : cInputInfo) {
+                                ptrInputBlobs.push_back(inferRequest.inferRequest.get_tensor(input));
+                            }
+                        } else {
+                            std::vector<std::string> inputNameBlobs = convert_str_to_vector(FLAGS_iname);
```

Comment:
@mgrdv We need to check that all inputs are listet in iname. Later we should remove iname parameter and just introduce another format for -i parameter `<input_name1>:<file1>,<input_name2>:<file2>`